python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -690,11 +689,17 @@ def _flush(self, objs):
             if not obj._dirty:
                 continue
 
+            if hasattr(obj, _reg_id_str):
+                obj_id = getattr(obj, _reg_id_str)
+                if obj_id not in self._loaded_ids:
+                    continue
+            else:
+                continue
```

Comment:
It's a belt and braces check and it's skipping any job which is not in the ```_loaded_ids``` list, this is what the code does so I fail to see how a comment helps?